code/game/machinery/computer/HolodeckControl.dm

```diff
@@ -407,10 +407,11 @@
 /turf/simulated/floor/holofloor/grass
 	name = "lush Grass"
 	icon_state = "grass1"
+
+/turf/simulated/floor/holofloor/grass/create_floor_tile()
 	floor_tile = new/obj/item/stack/tile/grass
 
 /turf/simulated/floor/holofloor/grass/New()
-	floor_tile.New() //I guess New() isn't run on objects spawned without the definition of a turf to house them, ah well.
```

Comment:
What did he mean by this?

Reply:
Might have been a holdover from the dark ages during which `getFromPool` didn't call New

Reply:
How could `getFromPool()` have been called before this line executes though?

Reply:
we just don't know

Reply:
I actually just checked blame and this particular line *predates pooling*